docs/Installation.md

```diff
@@ -3,6 +3,9 @@ title: Installation
 permalink: /install/
 ---
 
+## Warning
+**Currently react-native-track-player does not support projects with Expo or Expokit.**
+
```

Comment:
I think that shouldn't be a warning, but a note. Perhaps it should belong to the troubleshooting section instead.